src/features/currentTodo.ts

```diff
@@ -1,34 +1,31 @@
 import { Todo } from '../types/Todo';
 
-// we use string literal as a type to avoid mistype in future
 type RemoveTodoAction = { type: 'currentTodo/REMOVE' };
 
-// payload is a typical name for an action data
 type SetTodoAction = {
   type: 'currentTodo/SET';
```

Comment:
Just compare this action type (it has a prefix) and yours (it doesn't). Consider standardization.